run-custom-mao.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash -xe
+
+CUSTOM_MAO_IMAGE=${CUSTOM_MAO_IMAGE:-$1}
+if [ -z "$CUSTOM_MAO_IMAGE" ]; then
+    # Custom MAO image not already provided
+    echo "Usage: ./run-custom-mao.sh <machine-api-operator image> <repo name> <branch name>"
+    echo "Machine-api-operator image to be tested is mandatory."
+    echo "repo_name and branch_name are optional parameters."
+    echo "If these are not provided, they will default to "openshift" and "master" respectively."
```

Comment:
If you want the quotes around openshift/master to be displayed in the output they need to be escaped.